src/main/java/com/google/sps/data/AccessSecret.java

```diff
@@ -0,0 +1,34 @@
+package com.google.sps.data;
+
+import com.google.cloud.secretmanager.v1.AccessSecretVersionResponse;
+import com.google.cloud.secretmanager.v1.SecretManagerServiceClient;
+import com.google.cloud.secretmanager.v1.SecretVersionName;
+import java.io.IOException;
+
+public class AccessSecret {
```

Comment:
Since your key doesn't change, perhaps this class could follow the singleton pattern around the key? If the key isn't set when called, make the call to the managerclient, and then return it. Of course don't forget to add validations and have error handling.

Reply:
done